cli/src/commands/v2-migrations/20171122100800_common_indexes.js

```diff
@@ -29,10 +29,9 @@ const createStatementIndexes = (connection) => {
   createIndexWithVoid('verbId_objId', { ...verbIdIndex, ...objectIdIndex });
   createIndexWithVoid('mbox', { 'statement.actor.mbox': 1 });
   createIndexWithVoid('account', accountIndex);
-  createIndexWithVoid('timestamp__id', { timestamp: -1, _id: -1 });
   createIndexWithVoid('stored__id', { stored: -1, _id: -1 });
   createIndex(stmts, 'statementId_lrs_id', { 'statement.id': 1, lrs_id: 1 });
-  createIndexWithOrg('timestamp__id', { timestamp: -1, _id: 1 });
+  createIndexWithOrg('timestamp__id', { timestamp: -1, _id: -1 });
```

Comment:
Added an amalgamation of the two - it seems to ignore the order when deciding if it's a duplicate